GameServerCore/Domain/GameObjects/IObjAiBase.cs

```diff
@@ -20,28 +20,16 @@ public interface IObjAiBase : IAttackableUnit
         bool IsMelee { get; set; }
         MoveOrder MoveOrder { get; }
         IAttackableUnit TargetUnit { get; set;  }
+        IBuffManager Buffs { get; }
```

Comment:
Maybe we should move this to IAttackableUnit so Turrets and other attackable objects can get buffs.